modules.d/91tpm2-tss/module-setup.sh

```diff
@@ -52,6 +52,7 @@ install() {
         {"tls/$_arch/",tls/,"$_arch/",}"libtss2-tcti-swtpm.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libtss2-tctildr.so.*" \
         {"tls/$_arch/",tls/,"$_arch/",}"libcryptsetup.so.*" \
+        {"tls/$_arch/",tls/,"$_arch/",}"libcryptsetup-token-systemd-tpm2.so.*" \
```

Comment:
Will this pick up the library from `/usr/lib/cryptsetup/`?  That's were the file is on my Arch system…

I'm sorry if that's a stupid question but I'm not familiar with dracut :slightly_smiling_face:

Reply:
Added the directory path. 

I just assumed that distro's would use --with-luks2-external-tokens-path set to lib since to me, using that cryptsetup directory makes no sense since it configurable --with-luks2-external-tokens-path

Reply:
Thanks.  :pray: